angular-workspace/projects/example-client-app/src/app/customapp/customapp.component.html

```diff
@@ -202,6 +202,17 @@
                 </nimble-tab-panel>
             </nimble-tabs>
         </div>
+        <div class="sub-container">
+            <div class="container-label">Tabs - Anchor</div>
+            <nimble-anchor-tabs #anchorTabs activeid="1">
+                <nimble-anchor-tab id="1" href="#" (click)="onAnchorTabClick('1')">Tab 1</nimble-anchor-tab>
+                <nimble-anchor-tab id="2" href="#" (click)="onAnchorTabClick('2')">Tab 2</nimble-anchor-tab>
+                <nimble-anchor-tab disabled id="3" href="#" (click)="onAnchorTabClick('3')">Tab 3 (Disabled)</nimble-anchor-tab>
+                <nimble-tabs-toolbar>
+                    <nimble-button (click)="onTabToolbarButtonClick()">Toolbar button</nimble-button>
+                </nimble-tabs-toolbar>
+            </nimble-anchor-tabs>
```

Comment:
We should keep the example as in-sync with the blaor example as possible. So we should add the select used to set activeid to this example and the tabs example (or remove them from both): see https://github.com/ni/nimble/pull/989#discussion_r1084679097

Reply:
Done